src/worker/util/achievement.ts

```diff
@@ -131,17 +131,29 @@ async function getAll(): Promise<
 
 const check = async (when: AchievementWhen, conditions: Conditions) => {
 	try {
+		/* took out this early return to explicitly let users know when achievements arent awarded.
```

Comment:
Just delete this rather than commenting it out. In general it is best to keep old versions of your code in your git history, not in comments.